Packages/com.unity.inputsystem/InputSystem/Plugins/XR/GenericXRDevice.cs

```diff
@@ -48,7 +69,25 @@ public class XRController : TrackedDevice
         /// <summary>
         /// A quick accessor for the currently active left handed device.
         /// </summary>
-        /// <remarks>If there is no left hand connected, this will be null. This also matches any currently tracked device that contains the 'LeftHand' device usage.</remarks>
+        /// <remarks>
+        /// If there is no left hand connected, this will be null.
+        /// This also matches any currently tracked device that contains the 'LeftHand' device usage.
+        /// <example>
+        /// <code>
+        /// // To set up an Action to specifically target
+        /// // the left-hand XR controller:
+        /// 
+        /// var action = new InputAction(binding: "/&lt;XRController&gt;{leftHand}/position");
+        /// </code>
+        /// 
```

Comment:
Seems to be missing a closing </example> here

Reply:
fixed